arduino-hal/src/port/feather_32u4_rfm.rs

```diff
@@ -0,0 +1,152 @@
+pub use atmega_hal::port::mode;
+pub use atmega_hal::port::Pin;
+
+avr_hal_generic::renamed_pins! {
+    type Pin = Pin;
+
+    /// Pins of the **Adafruit Feather 32u4 RFM69 and RFM9* **.
+    ///
+    /// This struct is best initialized via the [`arduino_hal::pins!()`][pins] macro.
```

Comment:
```suggestion
    /// This struct is best initialized via the [`arduino_hal::pins!()`][crate::pins] macro.
```

(change from f7680d6f564bc4bc753c9368ca15d4a376338741).